release_images.yml

```diff
@@ -80,3 +80,51 @@ release_images:
       # to images being published.
       force_release: False   # [Default: False] Set to True to force images to be published even if the same image
       # has already been published. Re-released image will have minor version incremented by 1.
+  5:
+    framework: "tensorflow"
+    version: "1.15.5"
+    arch_type: "x86"
+    inference:
+      device_types: [ "neuron" ]
+      python_versions: [ "py37" ]
+      neuron_sdk_version: "sdk1.17.1"
+      os_version: "ubuntu18.04"
+      example: False
+      disable_sm_tag: True
+      force_release: False
+  6:
+    framework: "tensorflow"
+    version: "2.5.3"
```

Comment:
Please change this to 2.5.2

Reply:
The https://github.com/aws/deep-learning-containers/blob/master/tensorflow/buildspec-neuron.yml file tags the image as TF 2.5.3. In that case, should we rebuild the RCs with the appropriate tags?